doc/source/commands/communication.rst

```diff
@@ -170,23 +170,37 @@ you want to use.
     currently available (because the required mod isn't installed, or the system
     is disabled in the settings) it will not be shown in lists of available managers.
 
-    StockConnectivityManager
-      This manager will ignore all connectivity limitations.  Connections
+    PermitAllConnectivityManager
+      This manager will permit all connectivity at all times.  Connections
       between vessels, home, and control always show as being connected. This
       is the equivalent of setting ``CONFIG:RT`` to ``False`` under the former
       system. It is possible kOS will be unable to use all features if this
       manager is selected while communication limitations are enforced by
       another mod or a setting.
     CommNetConnectivityManager
       This manager will use KSP's stock CommNet implementation to monitor
-      connections.  CommNet has limitations on updating connections for vessels
-      which are not the active vessel.  The best way to ensure that a connection
-      is updated is to include a relay antenna on one or both of the vessels.
+      connections.  It will only be available if CommNet is enabled in the KSP
+      difficulty settings.
+
+      .. note::
+
+          CommNet has limitations on updating connections for vessels
+          which are not the active vessel.  The best way to ensure that a connection
+          is updated is to include a relay antenna on one or both of the vessels.
```

Comment:
"a relay antenna" -> would be more  clear if it was phrased as "one of the kinds of antenna that can act as a relay".  Phrasing it that way makes it very clear that we are drawing a distinction between two types of antenna, (as opposed to "relay antenna versus no antenna")

If you don't mind, I'll make the edit as I merge (I hope such a small change is okay with you.  If not we can back it out.  I'd rather not go back and forth again and again multiple times on this and just hurry up and get it merged.)